webapp/src/main/java/com/box/l10n/mojito/service/thirdparty/ThirdPartyTMSSmartling.java

```diff
@@ -143,6 +132,36 @@ public ThirdPartyTMSSmartling(
     this.assetTextUnitToTMTextUnitRepository = assetTextUnitToTMTextUnitRepository;
   }
 
+  @Override
+  public void removeImage(String projectId, String imageId) {
```

Comment:
Can we have some integration test please? ThirdPartyTMSSmartlingITest or ThirdPartyTMSSmartlingTest